mavros/bdda1fa/bdda1fa.bug

```diff
@@ -81,4 +81,4 @@ fault-codes:
   - EVOLUTION:PACKAGE
 failure-codes:
   - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
Is this component used for operators to provide commands to the drone?

Reply:
Yes